Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: build run
+build:
+	docker run --rm -v `pwd`:/go/src/go-filter -w /go/src/go-filter \
+		-e GOPROXY=https://goproxy.cn \
+		golang:1.19.8 \
+		go build -v -o libgolang.so -buildmode=c-shared .
+
+run:
+	docker run --rm -v `pwd`/envoy.yaml:/etc/envoy/envoy.yaml \
+		-v `pwd`/libgolang.so:/etc/envoy/libgolang.so \
+		-p 10000:10000 \
+		-e GODEBUG=cgocheck=0 \
+		envoyproxy/envoy:contrib-v1.26-latest \
+		envoy -c /etc/envoy/envoy.yaml
+
+test:
```

Comment:
It would be great to rewrite this stuff into a Go integration framework. @doujiang24 What do you think?

Reply:
yep, that would be great. and here is an example: https://github.com/mosn/envoy-go-ldap-auth/tree/main/test

Reply:
ok, I will try this.

Reply:
Done.